[2024-11-02T13:31:02.754+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-05-07T00:00:00+00:00 [queued]>
[2024-11-02T13:31:02.766+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-05-07T00:00:00+00:00 [queued]>
[2024-11-02T13:31:02.767+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T13:31:02.795+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-05-07 00:00:00+00:00
[2024-11-02T13:31:02.801+0000] {standard_task_runner.py:57} INFO - Started process 3531 to run task
[2024-11-02T13:31:02.808+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-05-07T00:00:00+00:00', '--job-id', '1315', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmpsa5stslt']
[2024-11-02T13:31:02.812+0000] {standard_task_runner.py:85} INFO - Job 1315: Subtask submit_schemas
[2024-11-02T13:31:02.867+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-05-07T00:00:00+00:00 [running]> on host 5e812a6a7c02
[2024-11-02T13:31:02.971+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='minhtranquang' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-07T00:00:00+00:00'
[2024-11-02T13:31:02.973+0000] {pinot_schema_operator.py:33} ERROR - An error occured: 'function' object has no attribute 'glob'
[2024-11-02T13:31:02.982+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240507T000000, start_date=20241102T133102, end_date=20241102T133102
[2024-11-02T13:31:03.022+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T13:31:03.043+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
